src/Nulls.jl

```diff
@@ -42,19 +47,18 @@ Base.isless(::Null, b) = false
 Base.isless(a, ::Null) = true
 
 # Unary operators/functions
-for f in (:(+), :(-), :(*), :(/),
+for f in (:(+), :(-), :(Base.identity),
           :(Base.abs), :(Base.abs2), :(Base.sign),
           :(Base.acos), :(Base.acosh), :(Base.asin), :(Base.asinh), :(Base.atan), :(Base.atanh),
           :(Base.sin), :(Base.sinh), :(Base.cos), :(Base.cosh), :(Base.tan), :(Base.tanh),
           :(Base.exp), :(Base.exp2), :(Base.expm1), :(Base.log), :(Base.log10), :(Base.log1p),
           :(Base.log2), :(Base.exponent), :(Base.sqrt), :(Base.gamma), :(Base.lgamma),
-          :(Base.ceil), :(Base.floor), :(Base.round), :(Base.trunc)
-         )
+          :(Base.ceil), :(Base.floor), :(Base.round), :(Base.trunc))
     @eval $(f)(d::Null) = null
 end
 
 for f in (:(Base.iseven), :(Base.ispow2), :(Base.isfinite), :(Base.isinf), :(Base.isodd),
-          :(Base.isinteger), :(Base.isreal), :(Base.isempty))
+          :(Base.isinteger), :(Base.isreal), :(Base.isempty), :(Base.isnan), :(Base.iszero))
```

Comment:
Yep, just realized that. Also realized that `iszero` is a 0.6 thing, not 0.5, so I removed that.